.github/workflows/ci.yml

```diff
@@ -109,12 +109,13 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
 
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: ${{ matrix.python }}
```

Comment:
This is safe to do since 3.7-dev and so on will just point to the latest stable release that has been made, once a stable release has been made.